packages/services/src/vckit.service.ts

```diff
@@ -0,0 +1,38 @@
+import { CredentialPayload, VerifiableCredential } from '@vckit/core-types';
+import { contextDefault, typeDefault } from './models/vckit';
+import { publicAPI } from './utils/httpService';
+
+/**
+ * integrate with vckit to issue vc with default context and type
+ */
+export const integrateVckitIssueVC = async ({
+  context,
+  type,
+  issuer,
+  credentialSubject,
+  restOfVC,
+}: CredentialPayload): Promise<VerifiableCredential> => {
+  const body = constructCredentialObject({ context, type, issuer, credentialSubject, restOfVC });
+
+  try {
+    const response = await publicAPI.post(`${process.env.REACT_APP_VCKIT_API_URL ?? ''}/credentials/issue`, body);
```

Comment:
process.env.REACT_APP_VCKIT_API_URL, the variable will be imported from the configuration, so it should be a parameter of function